### ISSUE TITLE
Row repeating functionality is absent in Grafana 5.0

### ISSUE THREAD

**Issue by PhantomPhreak** [grafana/grafana#11650]
As i can see in Grafana 5.0.x release [notes](https://community.grafana.com/t/release-notes-v5-0-x/5250)
> Rows are not needed for layouts anymore and are mainly there for backward compatibility.

There are few problems related to this "feature":
1. Wrong [documentation](http://docs.grafana.org/reference/templating/#repeating-rowsl)
You can't set row repeating on new dashboards because there is no rows anymore.

2. Loss of functionality
You can't set row repeating on new dashboards because there is no rows anymore. And it's really bad, because rows repating is very useful feature.
Example: I have 5 host and 20 metrics for them.
Using rows repeating i can repeat rows for every selected host. For each host i can repeat panels in a row for selected metrics. It's very flexible and useful.

Is there any way to use similar way of panels oraganizing using Grafana 5.0.x? Only one way that i found - create dashboard using Grafana 4.x.x, export and import to Grafana 5.0.x, but it's really ugly way.

**Comment by kerams** [comment]
There is a problem with this "issue":

1. Not getting the facts right (`You can't set row repeating on new dashboards because there is no rows anymore.`)
You can add a row from the `Add panel` menu (granted, not the first place to look for it) and set up the desired row repeat in its options.

**Comment by PhantomPhreak** [comment]
Sorry, my bad, i didn't noticed it because it's in "Add panel" manu, and i expected that there is only panels.
Also it's quite confusing release notes, after loosing usual row controls it looks like "we removed them". I used to control rows via dashboard settings, but there is no rows-related settings also.

Anyway, thanks for making it clear, issue can be closed now.
I hope futher UI versions will have more obvious ways to add rows to dashboard.

Thanks a lot!
